File: smp2017-web-Rohit/CS50/PSET-1/mario.c
#include<stdio.h>
#include<cs50.h>
int main(void)
{
abc:printf("Enter a non negative number not greater than 23: ");
    int n=get_int();
    if((n<0)||(n>23))
    {
        printf("Error. Try again.\n");
        goto abc;
    }
    for(int i=1;i<=n;i++)
    {
        for(int j=(n-i);j>0;--j)
        {
            printf(" ");
        }
        for(int k=1;k<=i;k++)
        {
            printf("#");
        }
        printf("  ");
        for(int l=1;l<=i;l++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: smp2017-web-Rohit/CS50/PSET-6/mario.py
import cs50

# while True:
print("Enter a non negative number not greater than 23: ", end="")
n=cs50.get_int()
    # if n>0:
        # break;
        
for i in range(1,n+1):
    for j in range(n-i,0,-1):
        print(" ", end="")
    for k in range(1,i+1):
        print("#", end="")
    print("  ",end="")
    for l in range(1,i+1):
        print("#",end="")
    print("\n",end="")
        
            
